docs: added additional documentation for azure devops git provider (#6923)
coder/coder: docs/admin/git-providers.md

CODER_GITAUTH_0_TOKEN_URL="https://github.example.com/login/oauth/access_token"
```

> ### Azure Devops
> Azure Devops requires the following environment variables:

Reviewer: ```suggestion
### Azure DevOps
Azure DevOps requires the following environment variables:
```

```console
CODER_GITAUTH_0_ID="primary-azure-devops"
CODER_GITAUTH_0_TYPE=azure-devops
CODER_GITAUTH_0_CLIENT_ID=xxxxxx

> # ensure that this is the value for "Client Secret" and not "App Secret"

Reviewer: ```suggestion
# Ensure this value us your "Client Secret," not "App Secret"
```

CODER_GITAUTH_0_CLIENT_SECRET=xxxxxxx

> CODER_GITAUTH_0_AUTH_URL="https://app.vssps.visualstudio.com/oauth2/authorize"

Reviewer 2: @marleypowell thanks for your submission. can you update these URLs to something more generic? e.g. `https://azdevops.example.com`

Author: I'm not sure if that could potentially be misleading because ADO I think requires you to use these seemingly legacy urls.
